content/md/first-steps/manage-reference-entities.md

```diff
@@ -0,0 +1,233 @@
+:::
+
+:::tips
+To quickly translate all your options, you just have to translate the 1st label, then press `Enter`, translate the 2nd label, then press `Enter` and so on...
+:::
+
+
+### Remove options
+To remove an option, click on the cross at the end of the option line.
+
+## Define the completeness
+To define the attributes required for the completeness of a record:
+1.  Go to `Attributes` tab for a reference entity
+1.  Click on `Edit` icon for an attribute
+1.  Tick `Required for completeness` if this attribute is required for the completeness of a record
+
+![image](../img/ReferenceEntity_EditAttribute_Completeness.png)
+
+If this attribute is not required for the completeness, untick `Required for completeness`.
+
+## Delete an attribute
+To delete an attribute, follow these steps:
+1.  Go to `Attributes` tab for a reference entity
+1.  Click on `Edit` icon for the attribute you want to delete
+1.  An edit pop-in opens with the attribute parameters
+1.  Click on `Delete the attribute` button
+1.  A confirmation pop-in opens to confirm that you want to delete the attributes
+1.  Click on `Delete` button
+
+::: warning
+The attribute and its relevant information will be deleted in the records of the reference entity. Once you have confirmed the action, it cannot be reverted.
+:::
+
+# Delete a reference entity
+To delete a reference entity, follow these steps:
+1.  Go to `Properties` tab for the reference entity you want to delete
+1.  Click on `...` for other actions and then `Delete entity`
+1.  A confirmation pop-in opens to confirm that you want to delete the reference entity
+1.  Click on `Delete` button
+
+::: warning
+You cannot delete a reference entity if:
+- This reference entity has some records: you need to first [delete the records of this reference entity](/articles/enrich-your-reference-entity-records.md).
+- This reference entity is linked to some product attributes: you need to first [delete the related product attributes](/articles/manage-your-attributes.html#delete-an-attribute) to this reference entity.
+:::
+
+# Set permissions on a reference entity
+You can define which user groups can view or edit this reference entity, its properties, its attributes and its records.  
+The Edit right means add, edit and delete.
+
+:::info
+It's not possible to hide a reference entity.
+:::
+
+To define the permissions on a reference entity, follow these steps:
+1.  Go to `Permissions` tab of the reference entity
+1.  For each user group, tick to give the `View` or `Edit` rights
+1.  Click on `Save` button to save the Permissions
+
+![image](../img/ReferenceEntity_Permissions.png)
+
+For example, a user in the `Manager` group can edit this `Brand` reference entity, so add and edit attributes or add and enrich some brands.
+A user in the `Clothes managers` group only view this `Brand` reference entity, so only view its attributes and view the brands.
```

Comment:
Idem.

Reply:
+ "views" au lieu de "view" et je dirais : "so, he can only view its attributes and also view the brands", un truc du genre